lib/processor.js

```diff
@@ -12,6 +12,9 @@ var assign = require("object-assign");
 var remark = require("remark");
 
 var SUPPORTED_SYNTAXES = ["js", "javascript", "node", "jsx"];
+var UNSATISFIABLE_RULES = [
```

Comment:
That's a good point. I don't believe we have a way to support plugin-specific configuration. Is that correct, @ilyavolodin? If other core rules comes up, we can add them to this list. Hopefully by the time we get requests to exclude rules in third-party plugins, we have glob configs that will allow users to exclude those rules on their own.